app/services/AtsListService.scala

```diff
@@ -140,18 +141,18 @@ class AtsListService @Inject()(
       data.get
     }
 
-  private def sendAuditEvent(account: TaxIdentifier, dataOpt: Either[Int, AtsListData])(
+  private def sendAuditEvent(account: Option[TaxIdentifier], dataOpt: Either[Int, AtsListData])(
```

Comment:
Missing coverage for when account is `None`

Reply:
added test AtsListServiceSpec:473